packages/editor-ui/src/components/NodeCredentials.vue

```diff
@@ -101,15 +101,19 @@ export default mixins(
 		}),
 		isProxyAuth(): boolean {
 			return this.isHttpRequestNodeV2(this.node) &&
```

Comment:
considering product might have a few changes in the next versions... should we simply find a different way to flag http node as a proxy node? otherwise we have to touch the editor with every little tweak to http node.

Reply:
The only change to HTTPRN v2 that I can think of would require touching here would be renaming `existingCredentialType`, but that's true of `genericCredentialType` as well. So it's three conditions that have to be met for this node to be proxy auth. What did you have in mind?

Reply:
as simple as changing the default value for a param. I am very concerned about this..

Reply:
Changing the default value for a param in the HTTPRN is a breaking change and means creating a new version, so we'd have to edit the helper anyway. Maybe I don't understand what you mean by this?

Reply:
Addressed in refactoring branch: https://github.com/n8n-io/n8n/pull/3304